include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -811,9 +791,10 @@ namespace internal
       n_dof_indices(
         const dealii::DoFAccessor<structdim, dim, spacedim, level_dof_access>
           &                accessor,
-        const unsigned int fe_index_)
+        const unsigned int fe_index_,
+        const bool         level_dof_access_)
```

Comment:
Do get it right that you change this to be able to use this new parameter independently of the template parameter of `DoFAccessor`? Can you maybe add a more descriptive name, say `count_level_dofs` or something like that? (And the same in the function using this below.)

Reply:
Done :+1: